tests/unit/models/test_base_model.py

```diff
@@ -597,3 +599,34 @@ def test_to_xrpl_signer(self):
             ],
         }
         self.assertEqual(tx.to_xrpl(), expected)
+
+    def test_to_from_xrpl_xchain(self):
+        tx_json = {
+            "Account": account,
+            "Amount": value,
+            "XChainBridge": {
+                "LockingChainDoor": "rGzx83BVoqTYbGn7tiVAnFw7cbxjin13jL",
+                "LockingChainIssue": "XRP",
+                "IssuingChainDoor": "r3kmLJN5D28dHuH8vZNUZpMC43pEHpaocV",
+                "IssuingChainIssue": "XRP",
+            },
+            "Destination": destination,
+            "TransactionType": "XChainClaim",
+            "Flags": 0,
+            "SigningPubKey": "",
+            "XChainClaimID": "0000000000000001",
+        }
+        tx_obj = XChainClaim(
+            account=account,
+            amount=value,
+            xchain_bridge=XChainBridge(
+                locking_chain_door="rGzx83BVoqTYbGn7tiVAnFw7cbxjin13jL",
+                locking_chain_issue="XRP",
+                issuing_chain_door="r3kmLJN5D28dHuH8vZNUZpMC43pEHpaocV",
+                issuing_chain_issue="XRP",
+            ),
+            destination=destination,
+            xchain_claim_id="0000000000000001",
```

Comment:
Curiosity: Is "1" equivalent to "0000000000000001" in this case or no?

Reply:
UInts are encoded as hex strings, but I changed it to make it easier to read.